docs/rules/params-required.md

```diff
@@ -0,0 +1,32 @@
+# Parameters Required
+
+**Code**: params-required
+
+**Description**: Your template should have a parameters element. Parameters are essential for making your templates reusable in different environments. Add parameters to your template for values that change when deploying to different environments.
```

Comment:
parameters "element" is a bit confusing. In ARM templates this makes sense because it is a top-level property. How about:

>Your template should have at least one parameter.